src/main/scala/thurloe/service/ThurloeService.scala

```diff
@@ -20,6 +20,10 @@ trait ThurloeService extends HttpService {
   val dataAccess: DataAccess
   val thurloePrefix = "thurloe"
 
+  // TODO: Relies on swagger existing in resources/swagger/
+  val swaggerSite = path("swagger-site") { getFromResource("swagger/index.html") } ~ getFromResourceDirectory("swagger")
```

Comment:
In addition to @mcovarr's comment about moving yaml file to match the the endpoint, I don't believe for https://swagger.dsde-dev.broadinstitute.org we need this particular "swagger-site" endpoint, nor the `"org.webjars" % "swagger-ui" % "2.1.1"` in thurloe's build.sbt, only the thurloe.yaml file. In fact, a remaining cromwell TODO is to remove `swagger-ui` and `spray-swagger` now that the yaml appears to be working.